tablepyxl/style.py

```diff
@@ -172,8 +172,10 @@ def __init__(self, table):
         takes an html table object (from BeautifulSoup)
         """
         super(Table, self).__init__(table)
-        self.head = TableHead(table.thead, parent=self) if table.thead else None
-        self.body = TableBody(table.tbody or table, parent=self)
+        table_head = table.find('thead')
+        self.head = TableHead(table_head, parent=self) if table_head is not None else None
```

Comment:
It looks like this is already fixed on the master branch. https://github.com/martsberger/tablepyxl/pull/10/files

I don't know why we committed directly to master instead of develop and never pulled master back into develop.